tests/acceptance/30_generic_methods/file_ensure_key_value_present_in_ini_section.cf

```diff
@@ -57,7 +57,7 @@ ${base_text_up}";
     "name_3"            string => "name";
     "value_3"           string => "value";
     "new_value_3"       string => "newvalue";
-    "reference_3"       string => "${base_text_up}
+    "reference_3"       string => "[${section_3}]
```

Comment:
why this change?

Reply:
for readability

Reply:
i scratched my head thrice over the test of why it was section1 in this test, so had to look up to base_text_up, which is not informative